src/main/scala/scorex/transaction/assets/exchange/Order.scala

```diff
@@ -167,7 +166,7 @@ case class Order(@ApiModelProperty(dataType = "java.lang.String") senderPublicKe
     }
   }
 
```

Comment:
What's wrong with equals? Currently, fields, which are used in equals also used to create idStr hence hashcode